client/performance-profiler/components/core-web-vitals-display/index.tsx

```diff
@@ -43,6 +48,24 @@ export const CoreWebVitalsDisplay = ( props: PerformanceMetrics ) => {
 		return '';
 	};
 
+	const { performanceReport } = props;
+	let metrics: number[] = performanceReport?.history?.metrics[ activeTab ] ?? [];
+	let dates = performanceReport?.history?.collection_period ?? [];
+
+	// last 8 weeks only
+	metrics = metrics.slice( -8 );
+	dates = dates.slice( -8 );
+
+	// the comparison is inverse here because the last value is the most recent
+	const positiveTendency = metrics[ metrics.length - 1 ] < metrics[ 0 ];
+
+	const historicalData = metrics.map( ( item, index ) => {
+		return {
+			date: dates[ index ],
```

Comment:
I think an important note here, that we don't have the year in the API response:
```
"collection_period": [
"2/17",
"2/24",
"3/2",
"3/9",
"3/16",
"3/23",
"3/30",
"4/6",
"4/13",
"4/20",
"4/27",
"5/4",
"5/11",
"5/18",
"5/25",
"6/1",
"6/8",
"6/15",
"6/22",
"6/29",
"7/6",
"7/13",
"7/20",
"7/27",
"8/3"
],
```
If the historical data is between years and the month turns over, this could cause a problem when rendering the chart. Eg. rendering the chart with the following test data:
```
	const historicalData = [
		{ date: '9/17', value: 0.79 },
		{ date: '10/18', value: 0.8 },
		{ date: '11/19', value: 0.81 },
		{ date: '12/20', value: 1.79 },
		{ date: '1/21', value: 1.8 },
		{ date: '2/22', value: 1.81 },
		{ date: '3/23', value: 1.9 },
		{ date: '4/24', value: 1.91 },
	];
```
![CleanShot 2024-08-15 at 13 02 10@2x](https://github.com/user-attachments/assets/f4a52c5b-0df0-4e2e-801e-0b3f52f0600e)

To fix this, I think we'll need to add back the year on the backend here: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Sfvgr%2Qcebsvyre%2Sgenafsbezref%2Spynff%2Qcntrfcrrq%2Qgenafsbezre.cuc%3Se%3Q0o916188%2391-og